client/src/components/common/Image/index.js

```diff
@@ -0,0 +1,41 @@
+import Image from 'material-ui-image';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CommonImage = ({
+  src,
+  width,
+  height,
+  animationDuration,
+  onClick,
+  alt,
+  style,
+}) => (
+  <Image
+    imageStyle={{ width, height }}
+    animationDuration={animationDuration}
+    onClick={onClick}
+    alt={alt}
+    style={style}
+    src={src}
+  />
+);
+
+CommonImage.propTypes = {
+  src: PropTypes.string.isRequired,
+  width: PropTypes.string,
+  height: PropTypes.string,
+  animationDuration: PropTypes.number,
+  onClick: PropTypes.func,
+  alt: PropTypes.string,
+  style: PropTypes.string,
```

Comment:
please arrange proptypes by the same way in all places & put the `src` at the first in all cases.